File: Laboratorios C++/EvaluacionEstudiante2/EvaluacionEstudiante2.cpp
#include <iostream>
using namespace std;

int main()
{
    double nota;
    cout << "Ingrese la nota del estudiante (0-20): ";
    cin >> nota;

    if(nota >= 11)
    {
        cout << "El estudiante aprobó la prueba." << endl;
    }

    else
    {
        cout << "El estudiante no aprobó la prueba." << endl;
    }
    return 0;
}

File: Laboratorios Java/EvaluacionEstudiante2/EvaluacionEstudiante2.java
import java.util.Scanner;

public class EvaluacionEstudiante2
{
    public static void main(String[]args)
    {
        Scanner sc = new Scanner(System.in);
        System.out.print("Ingrese la nota del estudiante (0-20): ");
        double nota = sc.nextDouble();

        if(nota >= 11)
        {
            System.out.println("El estudiante aprobó la prueba.");
        }
        else
        {
            System.out.println("El estudiante no aprobó la prueba.");
        }
        sc.close();
    }
}